packages/cli/src/WorkflowExecuteAdditionalData.ts

```diff
@@ -702,7 +702,7 @@ export async function getRunData(
 
 export async function getWorkflowData(
 	workflowInfo: IExecuteWorkflowInfo,
-	parentWorkflowId?: string,
+	parentWorkflowId: string,
```

Comment:
To confirm my understanding - how do we know there's always a parent workflow ID? This is called by `WorkflowRunnerProcess.runWorkflow()` which might not be running a subworkflow?

Reply:
I also changed this in there to make it mandatory - all calls to `executeWorkflow` can only come from a subworkflow execution (this is the purpose of this function) so I made it be mandatory in `packages/workflow/src/Interfaces.ts` as part of `IWorkflowExecuteAdditionalData`